helpers/image_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    temp_file = Tempfile.new('images-', StaticData::TMP_FOLDER.to_s, encoding: 'ascii-8bit')
```

Comment:
> StaticData::TMP_FOLDER.to_s

Does temp file location is needed at all?
Default seems fine.
And default will work on both linux and windows